src/admin-views/tribe-options-display.php

```diff
@@ -42,66 +43,72 @@
 	],
-	'tec-events-settings-display-toc-calendar'         => [
+	'tec-events-settings-display-toc-calendar'   => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-events-settings-display-calendar">' . _x( 'Calendar Display', 'Calendar Display table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-events-settings-display-calendar">' . _x( 'Calendar Display', 'Calendar Display table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-events-settings-display-toc-date'         => [
+	'tec-events-settings-display-toc-date'       => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-events-settings-display-date">' . _x( 'Date & Time', 'Date & Time table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-events-settings-display-date">' . _x( 'Date & Time', 'Date & Time table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-events-settings-display-toc-currency' => [
+	'tec-events-settings-display-toc-currency'   => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-events-settings-display-currency">' . _x( 'Currency', 'Currency table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-events-settings-display-currency">' . _x( 'Currency', 'Currency table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-events-settings-display-toc-maps' => [
+	'tec-events-settings-display-toc-maps'       => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-events-settings-display-maps">' . _x( 'Maps', 'Maps table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-events-settings-display-maps">' . _x( 'Maps', 'Maps table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
 	'tec-events-settings-display-toc-additional' => [
 		'type' => 'html',
-		'html' => '<li><a href="#tec-settings-events-settings-display-additional">' . _x( 'Additional Content', 'Additional Content table of contents link.', 'the-events-calendar' ) . '</a>',
+		'html' => '<li><a href="#tec-settings-events-settings-display-additional">' . _x( 'Additional Content', 'Additional Content table of contents link.', 'the-events-calendar' ) . '</a></li>',
 	],
-	'tec-events-settings-display-toc-end'             => [
+	'tec-events-settings-display-toc-end'        => [
 		'type' => 'html',
 		'html' => '</ul>',
 	],
-	'tec-events-settings-display-toc-nav-end'                 => [
+	'tec-events-settings-display-toc-nav-end'    => [
 		'type' => 'html',
 		'html' => '</div>',
 	],
 ];
 
-$tec_events_display_fields += $tec_events_general_toc;
+/**
+ * Allow filtering the Table of Contents bar on the Display settings page.
+ *
+ * @var array $tec_events_display_toc Array of items of the TOC.
+ */
+$tec_events_display_toc = apply_filters( 'tec_events_display_settings_toc', $tec_events_display_toc );
```

Comment:
The docblock needs a `@since` and use `@param` folr filters, no need to filter into the variable, than apply the addition. Do it all in one line.